app/lib/theme/subthemes/icon_button_theme.dart

```diff
@@ -7,31 +7,29 @@ class FlexIconButtonTheme {
 
   static IconButtonThemeData lightIconButtonTheme = IconButtonThemeData(
     style: IconButton.styleFrom(
-      elevation: 1,
-      shadowColor: FlexAppColorScheme.lightScheme.primary,
-      foregroundColor: FlexAppColorScheme.lightScheme.primary,
+      elevation: 0,
```

Comment:
is this the z-stack?

Reply:
yep! removed it as it being set to `0` is redundant (default is `0`)